calico/_includes/charts/tigera-operator/Chart.yaml

```
@@ -1,6 +1,12 @@
 apiVersion: v2
 name: tigera-operator
 description: Installs the Tigera operator for Calico
-
+home: https://www.tigera.io/
+icon: https://tigera.io/wp-content/uploads/2021/05/cropped-tigera-icon-1.png
```

That's the Tigera icon, I think most people will be expecting the calico icon "Felix":
```suggestion
icon: https://projectcalico.docs.tigera.io/images/felix_icon.png
```